packages/components/package.json

```diff
@@ -64,13 +66,9 @@
     "postcss-import": "^12.0.1",
     "postcss-preset-env": "^6.7.0",
     "react-test-renderer": "^16.9.0",
-    "rollup": "^1.11.3",
-    "rollup-plugin-commonjs": "^9.3.4",
+    "rollup": "^2.79.1",
```

Comment:
yikes, you can really see how this was not upgraded in a long time 😬

Reply:
yeah, and they're on v3 now (just recently, but still lol)